src/providers/apolloClient.tsx

```diff
@@ -7,18 +7,13 @@ import {
 } from '@apollo/client';
 import { BatchHttpLink } from '@apollo/client/link/batch-http';
 import { setContext } from '@apollo/client/link/context';
-import { onError } from '@apollo/client/link/error';
-import * as Sentry from '@sentry/react';
+import { SentryLink } from 'apollo-link-sentry';
```

Comment:
@ttoomey this branch is up on QA right now, as you can see there is an issue with the syntax.

This package has [both cjs and esm exports](https://github.com/DiederikvandenB/apollo-link-sentry/blob/bcc93a12b2d3b63140f01e8aef7fb94100132a0e/package.json#L8-L11) and it seems like vite is pulling the cjs one (probably because its "main"). Not sure if there is a neater way to specify this, but it seems to work to replace this (and the other usage) with importing from `'apollo-link-sentry/lib-esm';`.

Reply:
I'm not sure if that will fix the issue, but might be worth a shot. I'm deploying a different commit to qa so its accessible again, but feel free to deploy this branch again if you need!

Reply:
That didn't fix it. It's weirdly trying to use "require" to load the sentryInit file. I wonder if this has something to do with the fact that the order of imports changed now that we call sentryInit first thing in the index file.

It's really weird that I can't reproduce this with `yarn build` -- when I run that locally, the index.js file does not use required. But the deployment leads me to believe its generating differently [when building in the container](https://github.com/greenriver/hmis-frontend/blob/a4a6045eac20f818bade8c3dcd717314439f42c1/docker/Dockerfile#L32)(?)

<img width="627" alt="Screenshot 2024-06-10 at 2 20 03 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/d81f96a2-5d7a-42ec-a5c8-c25d96a55ead">